tests/dummy/app/templates/index.hbs

```diff
@@ -75,7 +75,7 @@
       <table>
         <thead>
           <tr>
-            <th></th>
+            <th>&vArr;</th>
```

Comment:
Why this change?

Reply:
This is to fix the A11Y failing test. I'm not sure if there is a better way to fix it.

Reply:
This error?

```
Table header text should not be empty 
```

Why was Item considered empty?

Reply:
Yes, that one.